src/promptflow/promptflow/_cli/data/package_tool/MANIFEST.in.jinja2

```diff
@@ -0,0 +1 @@
+include {{ package_name }}/icons
```

Comment:
why do we need this file?

Reply:
If the user needs to release this package, those non-Python files need to be include in the package